automation_tools/__init__.py

```diff
@@ -1261,6 +1266,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         host=host,
         sam=distribution.startswith('sam'),
         sat_version=satellite_version,
+        sat_release=satellite_release,
```

Comment:
Is this prepared to receive `None` as release? Otherwise we should make sure a proper release is passed.